migrations.lock

```diff
@@ -343,6 +343,8 @@ enterprise
  0002_enterprisepermissions_account_unique
  0003_enterprisepermissions_modify_account
  0004_enterprisemobileworkersettings
+es
+ 0001_initial
```

Comment:
should this name reflect the actual migration name?

Reply:
Yeah. It should be, have updated it in https://github.com/dimagi/commcare-hq/pull/32393/commits/02977040008d2aa5032d6cb5028c8134a3ef2a4a